src/serlo-editor/plugins/multimedia/editor.tsx

```diff
@@ -53,24 +53,31 @@ export function MultimediaEditor(props: MultimediaProps) {
 
   return (
     <div className="group/multimedia">
-      {editable && focused ? (
-        <MultimediaToolbar
-          {...props}
-          showSettingsModal={showSettingsModal}
-          setShowSettingsModal={setShowSettingsModal}
-        />
-      ) : null}
-      {editable && !focused ? (
+      {editable && focused && (
```

Comment:
ah, missed that one. 
I always liked the brevity of this but got convinced by I think Jonas to always use the `isTrue ? <>…</> : null` pattern instead just to never run into annoying bugs ([some random article on it](https://www.crocoder.dev/blog/react-conditional-rendering/)). 
In this case it should never matter so I don't mind, but in general this is why I stick with the longer ternary.

Reply:
🐊

Reply:
Oh, good to know! I'll keep that in mind and try to stick to the longer version in the future!